Improved CSS for JoinPage, Preference and fixed warnings on console for lobby for frontend

DavidHuangz opened the pull request:
Improved CSS for JoinPage, Preference and fixed warnings on console for lobby. Added more css files etc.

Addresses issue #[165](https://github.com/we-are-number-1/yumble/issues/165)

OverCry reviewed (dismissed):
Looks good. Just not sure about 
![image](https://user-images.githubusercontent.com/54381704/111853743-2802f100-8981-11eb-95ee-26a684b0560c.png)
but it should be fine since it is empty

DavidHuangz commented:
> Looks good. Just not sure about
> ![image](https://user-images.githubusercontent.com/54381704/111853743-2802f100-8981-11eb-95ee-26a684b0560c.png)
> but it should be fine since it is empty

yea I deleted it. Fixed.

OverCry reviewed (dismissed):
Looks good!

OverCry reviewed (dismissed):
still looks good

ayushjani reviewed (approved):
LGTM

DavidHuangz commented:
Rip, need another two quick review

ayushjani reviewed (approved):
Still looks good 👍

jemimahmartinez commented on frontend/src/components/2.Join/JoinGroup.js:
```diff
@@ -33,33 +34,33 @@ function GroupCode() {
                 type='text'
                 inputType='joinGroup'
-                placeholder='e.g P6aPE'
+                placeholder=' P6aPE'
```
Why did you take out the 'e.g.'? I feel like it might help the user?

DavidHuangz replied:
I feel without the e.g looks better and also on many websites I've been to, they don't include that in the placeholder. But I can change it back.

jemimahmartinez replied:
Aw nah I think thats alg, I took a look at it properly when I ran it and it looks better actually

DavidHuangz replied:
yea, feel free to change it on your next commit if you feel like it might look better. :)

OverCry reviewed (approved):
Still looks good

jemimahmartinez reviewed (approved):
LGTM 👌
